README.md

```diff
@@ -2,12 +2,16 @@
 
 [![Build Status](https://cavertes.visualstudio.com/VW_Sharing/_apis/build/status/aci-privateagents-kv-integration?branchName=master)](https://cavertes.visualstudio.com/VW_Sharing/_build/latest?definitionId=3&branchName=master)
 
+This repository shows how to build a scalable pool of Azure DevOps ACI-based agents that have private access to a secured Keyvault to retrieve secrets (in this case, an SSH private key) to operate over air-gapped resources in Azure. 
 
-This repo shows how to build a scalable pool of Azure DevOps ACI-based agents that have private access to a secured Keyvault to retrieve secrets (in this case, an SSH private key) to operate over air-gapped resources in Azure. 
+Particular attention has been placed on security: the example secrets generated and upload in the pipeline is further made unaccessible by the Azure DevOps Service Connection by locking down the Keyvault access policy.
```

Comment:
This sentence is worded in a confusing way. I suggest rewording it, here's my best attempt at that: Generating secrets and uploading them in the pipeline is not possible for the Azure DevOps Service Connection as it's blocked by the KeyVault Access Policy.

Reply:
@ams0 this is for you.

Reply:
addressed in #13

Reply:
and reverted this README to original from master